includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1016,20 +1047,29 @@ public function wc_setup_shipping_save() {
 		update_option( 'woocommerce_weight_unit', $weight_unit );
 		update_option( 'woocommerce_dimension_unit', $dimension_unit );
 
+		$setup_wcs_labels  = isset( $_POST['setup_woocommerce_services'] ) && 'yes' === $_POST['setup_woocommerce_services'];
+		$setup_shipstation = isset( $_POST['setup_shipstation'] ) && 'yes' === $_POST['setup_shipstation'];
+
+		if ( $setup_wcs_labels ) {
+			$this->install_woocommerce_services();
+		}
+
+		if ( $setup_shipstation ) {
+			$this->install_plugin(
+				'woocommerce-shipstation',
+				array(
+					'name'      => __( 'ShipStation', 'woocommerce' ),
+					'repo-slug' => 'woocommerce-shipstation',
```

Comment:
This doesn't seem to be a valid slug on .org, and the plugin doesn't install when I try to install it from the wizard. Is it not live on .org yet? If not, when will it be?

Reply:
It's not live on .org yet. Ideally we will have it up by the time the RC is cut - still need to work that out with @laurendavissmith.